### ISSUE TITLE
UnrecognizedPropertyException for PlayerPosition telemetry events

### ISSUE THREAD

**Issue by Tetraquark** [kevcodez/pubg-api-kotlin#11]
Hey, it's me again :smile: 

Got an UnrecognizedPropertyException when trying to get telemetry:

```
com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "minFPS" (class de.kevcodez.pubg.model.telemetry.events.PlayerPosition), not marked as ignorable (5 known properties: "character", "common", "_D", "elapsedTime", "numAlivePlayers"])
 at [Source: (StringReader); line: 1, column: 20998] (through reference chain: java.util.ArrayList[103]->de.kevcodez.pubg.model.telemetry.events.PlayerPosition["minFPS"])
...
at de.kevcodez.pubg.client.ApiClient.getTelemetryData(ApiClient.kt:105) ~[classes/:na]
```

You can check the match telemetry data here: https://telemetry-cdn.playbattlegrounds.com/bluehole-pubg/pc-ru/2018/07/20/16/53/65deb5b4-8c3d-11e8-ab1f-0a5864624422-telemetry.json 

There are new undocumented (at that moment) fields `minFPS` and `maxFPS` in the event LogPlayerPosition (model.telemetry.events.PlayerPosition class).

I think it's better to just add the annotation `@JsonIgnoreProperties(ignoreUnknown = true)` of Jackson FasterXML lib for each telemetry event class because API developers too often add different undocumented fields.

**Comment by kevcodez** [comment]
@Tetraquark Thanks for taking the time to report all the issues you find. I have to leave right now and will be back later on this evening. I'll fix it and release it when I'm back.

Indeed, ignoring unknowns fields will greatly reduce those errors.

**Comment by kevcodez** [comment]
Fixed and released version 0.3.2
